packages/common-styles/.npmignore

```diff
@@ -29,3 +29,8 @@ tsd.json
 tslint.json
 typings
 visualtests
+
```

Comment:
this package will also ship `/src`

<img width="606" alt="image" src="https://user-images.githubusercontent.com/1223799/236483846-e73c7981-7430-4144-9655-959120571c74.png">

looking at some older version it seems to be expected https://unpkg.com/browse/@fluentui/common-styles@1.2.7/src/ 👀